addons/block_code/blocks/math/randf_range.tres

```diff
@@ -10,7 +10,7 @@ description = "Generate a random floating point number between [i]from[/i] and [
 category = "Math"
 type = 3
 variant_type = 3
-display_template = "Random floating point number between {from: FLOAT} and {to: FLOAT}"
+display_template = "random floating point number between {from: FLOAT} and {to: FLOAT}"
```

Comment:
A question for another day, but, do students know what "floating point number" means? Would "real number" be better understood?

Reply:
I think that we should remove one of these random number blocks and have a single "random number between X and Y". What I'm not sure about is which one should survive, the one for floats or the one for ints. Let's do that as follow-up.